examples/doc-scan/views/pages/partials/check.ejs

```diff
@@ -1,5 +1,12 @@
-<% displayGeneratedProfile = typeof hasGeneratedProfile === 'undefined' ? false : hasGeneratedProfile %>
-<% displayWatchlistSummary = typeof hasWatchlistSummary === 'undefined' ? false : hasWatchlistSummary %>
+<%
+formatString = function (value) {
+    if(value) {
+        const stringValue = value.toString()
+        return stringValue.charAt(0).toUpperCase() + stringValue.slice(1).replace(/_/g, ' ').replace(/-/g, ' ').toLowerCase()
```

Comment:
So, it capitalises and replaces `_` and `-` with a space.
One could say that you could break it down in 2 functions, and if you need still have an extra function  which combines both.
Regarding the replace bit, you also combine into one call with the `|` operator, aka using `/_|-/g` for pattern.

Reply:
Addressed this one.